public/app/core/utils/object.ts

```diff
@@ -10,9 +10,9 @@ export function sortedDeepCloneWithoutNulls<T>(value: T): T {
       .sort()
       .reduce((acc: any, key) => {
         const v = (value as any)[key];
-        // Remove null values and also -Infinity which is not a valid JSON value
-        if (v != null && v !== -Infinity) {
-          acc[key] = sortedDeepCloneWithoutNulls(v);
+        // Remove null values and convert -Infinity to 0 because -Infinity is not a valid JSON value
+        if (v != null) {
+          acc[key] = v === -Infinity ? 0 : sortedDeepCloneWithoutNulls(v);
```

Comment:
The correct way to fix this issue is to make `value` optional. The 0 is added in the backend because the schema marks this as mandatory, and the default value for a number is 0.

Another option, if we want to mark it as required is to set a very small number as a default value, but this could be problematic between languages.